projects/packages/account-protection/src/class-email-service.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * Class used to define Email Service.
+ *
+ * @package automattic/jetpack-account-protection
+ */
+
+namespace Automattic\Jetpack\Account_Protection;
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+use Jetpack_Options;
+
+/**
+ * Class Email_Service
+ */
+class Email_Service {
+	/**
+	 * Send auth email.
+	 *
+	 * @param \WP_User $user The user.
+	 * @param string   $auth_code The authentication code.
+	 *
+	 * @return bool True if the email was sent successfully, false otherwise.
+	 */
+	public function send_auth_email( \WP_User $user, string $auth_code ): bool {
+		$wp_send = $this->wp_send_auth_email( $user, $auth_code );
+
+		if ( ! $wp_send ) {
+			return $this->api_send_auth_email( $user, $auth_code );
+		}
+
+		return true;
+	}
+
+	/**
+	 * Send the email using wp_mail().
+	 *
+	 * @param \WP_User $user The user.
+	 * @param string   $auth_code The authentication code.
+	 *
+	 * @return bool True if the email was sent successfully, false otherwise.
+	 */
+	private function wp_send_auth_email( \WP_User $user, string $auth_code ): bool {
+		$blog_name = esc_html( get_bloginfo( 'name' ) );
+		$blog_url  = esc_url( get_bloginfo( 'url' ) );
+
+		$subject = esc_html__( 'Verify your identity at Jetpack', 'jetpack-account-protection' );
+		$message = sprintf(
+			/* translators: 1: User login, 2: User URL, 3: Blog name, 4: Authentication code, 5: User URL, 6: Blog name */
```

Comment:
```suggestion
			/* translators: 1: User login, 2: Blog URL, 3: Blog name, 4: Authentication code, 5: Blog URL, 6: Blog name */
```

Also (and sorry for only thinking of this now) - do we need to supply the blog URL twice? couldn't we just pass it once and then use the same placeholder multiple times? 🤔 Same goes for the blog name.